Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM ubuntu:16.04
 
 RUN apt-get update
-RUN apt-get install -y llvm-6.0 clang-6.0 libclang-6.0-dev \
-    linux-headers-4.4.0-98-generic linux-headers-4.10.0-14-generic \
+RUN apt-get install -y llvm-8 clang-8 libclang-8-dev \
```

Comment:
I noticed that the dockerfile does basically the same command as the Makefile? Is this necessary or should we just let it happen in the Makefile? Seems like it will just do the same thing twice